### PR TITLE
Update top banner announcements

### PR DESCRIPTION
**Pull request by EmilMalanczak**

### CODE REVIEW

File: packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx [hunk 1]
```diff
@@ -60,6 +60,12 @@ export function SettingsDropdown({
           </SettingsDropdownItem>
         </Link>
 
+        <Link to="https://legacy-app.spark.fi/" external>
+          <SettingsDropdownItem>
+            <SettingsDropdownItem.Content>Legacy interface</SettingsDropdownItem.Content>
```
Comment (EmilMalanczak) [inline packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx:65]: wondering if that icon should be used 
image

Comment (yivlad) [reply packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx:65]: I guess it's alright. Terms of use above is also a link and doesn't use the icon

File: packages/app/src/ui/atoms/top-banner/TopBanner.tsx [hunk 2]
```diff
@@ -3,14 +3,10 @@ import { Link } from '../link/Link'
 export function TopBanner() {
   return (
     <div className="w-full bg-spark/40 p-0.5 text-center text-basics-black/70 text-sm sm:p-0 sm:text-base">
```
Comment (yivlad) [inline packages/app/src/ui/atoms/top-banner/TopBanner.tsx:5]: Should we change the background to some kind of blue?

File: packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx (hunk 1 above)
Comment (EmilMalanczak) [reply packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx:65]: ye meant both of them as they are external

File: packages/app/src/ui/atoms/top-banner/TopBanner.tsx (hunk 2 above)
Comment (krzkaczor) [reply packages/app/src/ui/atoms/top-banner/TopBanner.tsx:5]: Yes. Please use something out of the Sky brand colors. Pick from https://sky.money/ (maybe purple? maybe a quick gradient?)

File: packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx (hunk 1 above)
Comment (krzkaczor) [reply packages/app/src/features/navbar/components/settings-dropdown/SettingsDropdown.tsx:65]: I actually think that using that icon that you suggested makes sense. Can you also make ToS last link?

**Review by yivlad — APPROVED** [review]

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**